docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,40 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
+The following actions are available by right-clicking the project in the **Codewind Explorer** view. Most actions function only if the project is enabled.
 
-Some actions open the default browser. If you find that the default browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
-
-- **Open Application**: Opens the application in the default browser. This action is only available when the application is running or debugging.
-- **Project Overview**: Opens the overview page for this project. You can use this to see information about the project or edit project settings.
-- **Metrics Dashboard**: Opens the application monitor in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
-- **Performance Dashboard**: Opens the performance dashboard in the default browser. This action is available only when the application is running or debugging.
-- **Tekton Dashboard**: This option is available to you if you are using remote connections, and only if you installed pipelines. Use this option to open the dashboard.
-- **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
+Some actions open the default browser. If the default browser cannot handle the content, change the default browser.
 
+- **Open Application**: Opens the application in the default browser. This action functions only when the application is running or debugging. If you try to access it before running the application, a warning message appears.
+- **Project Overview**: Opens the overview page for the project. Use this action to see information about the project or edit project settings.
+- **Metrics Dashboard**: In Java, Node.js, and Swift applications, this action opens the Metrics Dashboard in the default browser. Use this action to monitor the activity and health of your application. This action is available only when the application is running or debugging.
+- **Performance Dashboard**: In Java, Node.js, and Swift applications, this action opens the Performance Dashboard in the default browser. This action functions only when the application is running or debugging. If you try to access it before running the application, a warning message appears.
+- **Open Container Shell**: Opens a shell into your application container. This action is available only when the container is active.
 - **Import Project**: Imports your project into the current workspace.
 - **Show Log Files**: If log files are available, this action displays a list of log files. Click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
```

Comment:
Show Log Files - If you are going to keep this, then you shouldn't be IDE-specific in "Show all logs" and "Hide all logs".

Reply:
We've been specifying where instructions differ according to IDE. For example, in this file, https://www.eclipse.org/codewind/remote-setupregistries.html, we say, "In Eclipse" and "In VS Code" at least until we can implement a better way to format the info, such as by adding tabs in the files to separate the IDEs.

That said, though, maybe we could say something like, "Show logs"/"Hide logs" so we're not using the exact wording of any one IDE?

Reply:
I agree been IDE-specific will be more useful for our users. The only thing is that:

```
Show Log Files: If log files are available, this action displays a list of log files. Click a log file to open it, or click Show All to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click Hide All to remove all log files from the Console view.
```

repeats what under:

```
- Show all logs: If log files are available, this action opens all available log files.
   - In Eclipse, this action is Show Log Files>Show All.
   - In Eclipse Che and VS Code, this action is Show all logs.
   - In IntelliJ, this action is Log Files>Show All.

- Hide all logs: Remove all log files from the Console view.
   - In Eclipse, this action is Show Log Files>Hide All.
   - In Eclipse Che and VS Code, this action is Hide all logs.
   - In IntelliJ, this action is Log Files>Hide All.
```

It would better that the first description for "Show Log Files" specifies that it is for Eclipse, and indicate below the two descriptions for "Show all logs" and "Hide all logs" the bullets for Eclipse Che, VS Code, and IntelliJ. As an additional note for the two last descriptions, you can indicate that users can toggle between logs using the drop-down menu that will appear above the logs' output:

<img width="837" alt="Screen Shot 2020-05-08 at 9 11 06 AM" src="https://user-images.githubusercontent.com/49531960/81408862-02278180-910c-11ea-8158-d054ce832b48.png">

<img width="629" alt="Screen Shot 2020-05-08 at 9 11 21 AM" src="https://user-images.githubusercontent.com/49531960/81408853-fe93fa80-910b-11ea-9b15-45e27c3f5772.png">

Reply:
Thanks for clarifying! I will try to reconcile "Show Log Files:" and "Show all logs:"/"Hide all logs:". :)